fstest/testserver/init.d/TestSwiftAIO

```diff
@@ -7,9 +7,15 @@ NAME=swift-aio
 . $(dirname "$0")/docker.bash
 
 start() {
+    # When running the second set of integration tests against SwiftAIO, we need to wait for the first run to stop.
+    if docker inspect ${NAME} >/dev/null 2>&1; then
+        echo "Waiting for the previous run to stop..."
+        sleep 3
+    fi
+
```

Comment:
This is needed between now we are running two integration test suites in a row. In my Linux vm it fails because docker is not able to remove the container from the first run in time:
```
    --- SKIP: TestIntegration/FsShutdown (0.00s)
=== RUN   TestIntegrationWithDirectoryMarkers
    fstests.go:423: Using remote "TestSwiftAIO:"
    fstests.go:433:
        	Error Trace:	/Users/joe.cai/git/rclone/fstest/fstests/fstests.go:433
        	            				/Users/joe.cai/git/rclone/backend/swift/swift_test.go:33
        	Error:      	Received unexpected error:
        	            	failed to run /Users/joe.cai/git/rclone/fstest/testserver/init.d/TestSwiftAIO start
        	            	docker: Error response from daemon: Conflict. The container name "/swift-aio" is already in use by container "d5b6cdaae65058d4159bf1f79013239d49ab910a5102df513762fab230dfef9e". You have to remove (or rename) that container to be able to reuse that name.
        	            	See 'docker run --help'.
        	            	: exit status 125
        	Test:       	TestIntegrationWithDirectoryMarkers
```